docs/source/conf.py

```diff
@@ -20,8 +20,8 @@
     "astropy": ("http://docs.astropy.org/en/stable/", None),
     "numpy": ("https://docs.scipy.org/doc/numpy/", None),
     "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
-    "matplotlib": ("http://matplotlib.org", None),
-    "sympy": ("https://docs.sympy.org", None),
+    "matplotlib": ("https://matplotlib.org", None),
+    "sympy": ("https://docs.sympy.org/latest", None),
```

Comment:
I don't see why this was required

Reply:
I noticed `http` was raising a warning in the RTD build process :)